gsa_admin.py

```diff
@@ -457,6 +457,56 @@ def syncDatabases(self, database_list):
       except:
         log.error("Unable to sync %s properly" % database)
 
+  def pauseCrawl(self):
+    """Pause crawl on the GSA.
+       Supports only 7.2 and higher.
+
+    Args:
+      None
+    """
+    self._login()
+    security_token = self.getSecurityToken('exportAllUrls')
+    if self.is72:
```

Comment:
Why do you go to extra lengths not to support 7.0? As far as I know, if you were to remove the check, it would just work.